pyproject.toml

```diff
@@ -66,6 +66,7 @@ litellm = ["litellm>=1.36.1"]
 llamaindex = ["llama-index>=0.10.35"]
 mistral0 = ["mistralai>=0.1.8,<1.0.0"]
 mistral1 = ["mistralai>=1.0.0"]
+scorers = ["openai>=1.0.0", "instructor>=1.5.2", "Levenshtein>=0.26.0", "google-generativeai>=0.8.0", "mistralai>=1.0.3", "anthropic>=0.30.0"]
```

Comment:
Do the scorers need all of these?  What if someone wants to use the scorers but is only using `openai`?

Reply:
hmmm we wanted scorers to support the 4 LLM providers. But @tcapelle maybe its fine to remove the LLM provider clients, they'll have to initialise a LLM client before initialising the scorer anyways, so they wouldn't be able to do that if they didn't have the lib installed?

Reply:
Yeah that would be ideal.

I'm thinking if a user is using openai and scorers, it would be
```
pip install weave[openai,scorers]
```

That means they don't have to install the other LLM providers and all the deps that come with those.

Reply:
dependencies now broken out into `scorers` and `scorers_tests`. `scorers` just installs instructor and levenshtein. The assumption is that the user will have the appropriate LLM client installed already,  otherwise they won't even be able to initialise a client to pass to the scorer.

The tests install instructor, levenshtein and also install the 4 LLM provider clients.